hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/hudi/HoodieWriterUtils.scala

```diff
@@ -84,6 +85,46 @@ object HoodieWriterUtils {
       HIVE_SYNC_USE_KERBEROS.key -> HIVE_SYNC_USE_KERBEROS.defaultValue,
       HIVE_SYNC_KERBEROS_PRINCIPAL.key -> HIVE_SYNC_KERBEROS_PRINCIPAL.defaultValue
     ) ++ DataSourceOptionsHelper.translateConfigurations(parameters)
+
+    val globalProps = DFSPropertiesConfiguration.getGlobalProps.asScala
```

Comment:
Not following this change. Why is L90-L127 needed?
I see that it's in your latest merge commit from master. But I don't see the same change in master.

Reply:
This is the information that I handled on the github interface, and the information that was written by the code conflict, github, I just resubmitted the code of the master branch and resubmitted it